### ISSUE TITLE
Missing Types - Pool Events

### ISSUE THREAD

**Issue by MatthewJohnSymons** [MariaDB/mariadb-connector-nodejs#86]
I am using Typescript and all of its glorious type checking with the mariadb package. When creating a pool and then wanting to listen for the four pool events, I get an error:

> Property 'on' does not exist on type 'Pool'.

Using the following code, mostly as per the docs but instead using import rather than require so that it picks up the declaration file:

```typescript
import mariadb from 'mariadb';
const pool = mariadb.createPool({ host: 'mydb.com', user:'myUser' });
pool.on('connection', (conn) => console.log(`connection ${conn.threadId} has been created in pool`);
```

Is there an issue with me or the module?

**Comment by rusher** [comment]
Right, Connector TypeScript definition is missing those pool event.  

This will be corrected in next version with https://github.com/MariaDB/mariadb-connector-nodejs/commit/34b3bbdc33c3a64e9919e6df466f930094f9a414

**Comment by rusher** [comment]
released in [2.1.4]( https://github.com/MariaDB/mariadb-connector-nodejs/releases/tag/2.1.4)
